src/data/fin.lean

```diff
@@ -148,6 +148,9 @@ iff.rfl
 @[norm_cast, simp] lemma coe_fin_le {n : ℕ} {a b : fin n} : (a : ℕ) ≤ (b : ℕ) ↔ a ≤ b :=
 iff.rfl
 
+lemma exists_eq_succ (a : fin n) : ∃ m, n = m + 1 :=
+nat.exists_eq_succ_of_ne_zero $ ((nat.zero_le _).trans_lt a.2).ne'
```

Comment:
This lemma seems very odd, given it uses the `a` argument as simply a proof that `n \ne 0`.

Reply:
Actually, I can just drop this lemma.